src/components/Common/Button.js

```diff
@@ -1,29 +1,14 @@
 import cx from "classnames";
 import "./Button.css";
 
-export default function Button({ href, imgSrc, children, onClick, className, size = "lg", align = "center", ...rest }) {
+export function ButtonWithExternalLink({ href, imgSrc, children, className, size = "lg", align = "center", ...rest }) {
```

Comment:
do you think it's necessary to rename button? i guess `Button` works as well? and it's shorted

also not sure why you removed `default`, i guess all components are exported with `default` so it's slightly shorted to import them. i guess it would be good to keep it the same for consistency?

Reply:
Yeah I just felt we are exporting multiple kinds of from this file. Like connect wallet, ButtonWithExternalLink and maybe in future we will add few more if needed.

The main reason behind renaming the Button to ButtonWithExternalLink is, earlier we were using it at multiple places like header, Buy page and other places. Now we only use at BuyGmx page.

Reply:
thank you

if i'm not wrong according to new structure/guides each component should have separate file anyway.